### ISSUE TITLE
Remove building=ruins preset

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/id-tagging-schema#832]
The tag `building=ruins` is highly controversial and [its wiki](https://wiki.openstreetmap.org/wiki/Tag:building%3Druins) even states that controversy.
The `building` tag should be used to define the types of buildings, and there are no buildings constructed to be ruins.
I've seen many wrong uses of this tag, propagated by this preset in iD.
Shouldn't it be removed?

Regards.

**Comment by 1ec5** [comment]
If it’s a valid tag but the preset is easily mistaken for something else, perhaps the preset’s name and/or icon should change to something less confusing?

**Comment by bryceco** [comment]
I think the preferred tagging is historic=ruins, ruins=building.

**Comment by matkoniecz** [comment]
> and there are no buildings constructed to be ruins

There are sham ruins, but typical `building=ruins` is not one of them.

**Comment by 1ec5** [comment]
By the way, the wiki’s definition is [itself disputed](https://wiki.openstreetmap.org/wiki/Talk:Tag:building%3Druins#Description_and_Usage_of_the_building.3Druins_tag). The contention in that discussion is that `building=ruins` can be used for (non-historic) buildings that weren’t constructed as ruins, since that’s how the tag has historically been used. My recollection is that this tag really took off after the earthquake in Haiti, when ruins were being identified systematically. If actual usage of this tag is more mixed than the wiki lets on, the wiki needs to make that clear so that mappers and data consumers don’t misinterpret existing usage.

**Comment by matkoniecz** [comment]
> My recollection is that this tag really took off after the earthquake in Haiti, when ruins were being identified systematically.

Yes, HOT has widely encouraged people to use `building=collapsed` `building=ruins` which is not really a correct tagging.

Not entirely sure whether we have established clear mapping for "there is a ruined building here and ruins are still present" that is not used also for other things.

**Comment by AntMadeira** [comment]
> I think the preferred tagging is historic=ruins, ruins=building.

I know `historic` tag as been (mis)used for many elements, but it is not required for a building in ruins. Historic means/should mean something important (historic) happened there. It is not necessary for a house where someone lived and is now in ruins.

What I see is that people want to map buildings (whatever the type of the building) that are in ruins, and they don't care if the tag is correct or not in iD. Since `building=ruins` is disputed (and, in my view, semantically incorrect), the iD preset should be adapted to correspond to those expectations. I'm sure that 99% of mappers who already used `building=ruins` never intended to map a "sham ruins".

**Comment by matkoniecz** [comment]
> Historic means/should mean something important (historic) happened there.

this is opinion held by someone about `historic=` but many tags like `historic=wayside_shrine` are not used in this way

**Comment by matkoniecz** [comment]
I started https://community.openstreetmap.org/t/do-we-have-building-for-existing-ruins-of-a-building/97276

**Comment by AntMadeira** [comment]
Sorry, I missed this reply somehow.
What I use is `building=yes|house|whatever` + `ruins=yes`.
But there's the alternative `ruins:building=yes`, which I don't like so much, because it' not rendered in the map.

Anyway, it doesn't really matter what's the alternative, because `building=ruins` is just plain wrong and should be removed as a preset.

> And which tagging would be better? Do we have one? Note that `demolished:building=` is also used for cases where both building and ruins are utterly gone

**Comment by mnalis** [comment]
> But there's the alternative `ruins:building=yes`

Yes, I'd say from that discussion that was the most accepted solution

>  which I don't like so much, because it' not rendered in the map.

[Do not tag for the renderer](https://wiki.openstreetmap.org/wiki/Tagging_for_the_renderer)... 

That being said, it renders nicely on _my_ preferred map  (e.g. this dashed-polygon in the middle of the screen is `ruins:building=yes` from here https://www.openstreetmap.org/way/90681114):

![Image](https://github.com/user-attachments/assets/286f0e77-5dea-496c-b0ec-e0a426a1c5e8)
